File: Assignment36_3.c
// Write a program which accept string from user and toggle the string
#include<stdio.h>

void toggle(char *str)
{
    while(*str != '\0')
    {

        if((*str >= 'a') && (*str <= 'z'))
        {
            *str = *str - 32;
            
        }
        else if((*str >= 'A') && (*str <= 'Z'))
        {
            *str = *str + 32;
        }
        str++;
    }
}

int main()
{
    char Arr[] = {'\0'};

    printf("Enter a String: \n");
    scanf("%[^'\n']s",Arr);

    toggle(Arr);
    
    printf("Updated string is: %s\n",Arr);

    return 0;
}

File: Assignment36_3.java
// Write a program which accept string from user and toggle the string
import java.util.*;

class Logic
{
    public void ToUpper(String str)
    {
        char Arr[] = str.toCharArray();
        int i = 0;

        for(i = 0; i < Arr.length; i++)
        {
            if((Arr[i]>= 'a') && (Arr[i] <= 'z'))
            {
                Arr[i] = (char)(Arr[i] - 32);
            }
            else if((Arr[i]>= 'A') && (Arr[i] <= 'Z'))
            {
                Arr[i] = (char)(Arr[i] + 32);
            }
        }

        String str1 = new String(Arr);

        System.out.println("Updated String is: "+str1);
    }

}


class Assignment36_3
{
    public static void main(String A[])
    {
        Scanner scanobj = new Scanner(System.in);

        System.out.println("Enter a string: ");
        String sobj = scanobj.nextLine();

        Logic lobj = new Logic();
        lobj.ToUpper(sobj);

    }
}